packages/react-cosmos-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cosmos-utils",
-  "version": "2.0.0-beta.1",
+  "version": "2.0.0-beta.2",
```

Comment:
No need to update versions. Lerna takes care of them automagically when running the publish cmd.